backend/reviews/templates/grant-review.html

```diff
@@ -112,7 +112,11 @@ <h2>Grant</h2>
 
   <div class="review-row">
     <strong>Grant type</strong>
-    <div>{{grant.get_grant_type_display}}</div>
+    <div>
+      {% for type_ in grant.grant_type %}
+        {{type_}}
```

Comment:
this will probably be displayed quite ugly?

Reply:
not too bad... 
![CleanShot 2024-12-01 at 22 35 17@2x](https://github.com/user-attachments/assets/7752a384-91a2-40ca-ab23-a81a83cbbb12)
![CleanShot 2024-12-01 at 22 35 09@2x](https://github.com/user-attachments/assets/4c0cdfc6-5bc1-4807-9f0e-ad02fe1a3015)